main.go

```diff
@@ -46,6 +54,10 @@ func init() {
 func main() {
 	flag.Parse()
 
+	if platform != "ubuntu" && platform != "windows" {
```

Comment:
The idea is:
* if ubuntu (or centos), report includes all the static info, plus the monit info
* if windows, report only includes the static info (but doesn't blow up trying to get monit info)
* otherwise, specified platform is invalid, blow up.